exporters/metrics/build.gradle

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+plugins {
```

Comment:
Does this belong in this PR?

Reply:
This is not related to fixing the underlying bug in any way. It could be added in a different PR, but since it was a small change I added it here. 

I noticed no downside of having this and this helps during manual testing by allowing the generation of a standalone JAR for this specific module via `./gradlew shadowJar` which can be used as a dependency.

Reply:
Consider using a different testing workflow, such as

* adding a new unit test within this project
* adding a new sub-project that depends on this project

If that test helps you locally, there's a good chance it would serve as a good demo/example too (so you might want to check it in). Using Gradle's project dependency instead of copying jars manually will also save you time and avoid mistakes from using stale jars.

Reply:
Removed the plugin since we already have examples sub-project that help with local testing.